src/tsolvers/lasolver/LABounds.cc

```diff
@@ -1,19 +1,19 @@
 #include "LABounds.h"
 #include "LRALogic.h"
 
-LABound::LABound(BoundT type, LVRef var, const Delta& delta, int id)
+LABound::LABound(BoundT type, LVRef var, Delta && delta, int id)
     : type(type.t)
     , bidx(UINT32_MAX)
     , id(id)
     , var(var)
     , delta(delta)
```

Comment:
To really avoid the copy, we need to change also this to `delta(std::move(delta))`, I believe.

Reply:
ok.